packages/cli/src/handlers/validate.ts

```diff
@@ -140,8 +146,8 @@ export const validateHandler = async (options: ValidateHandlerOptions) => {
         console.error(columns);
 
         console.error(
-            `\nFor more details, visit ${styles.bold(
-                `${config.context?.serverUrl}/generalSettings/projectManagement/${projectUuid}/validator`,
+            `\nTo see these errors in Lightdash, run ${styles.bold(
```

Comment:
I wonder if we can make this a bit more "stand-out"ish? Maybe adding an arrow at the start or something to draw people's attention to it? 

```suggestion
            `\n--> To see these errors in Lightdash, run ${styles.bold(
```